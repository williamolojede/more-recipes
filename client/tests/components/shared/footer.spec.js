import React from 'react';
import Footer from '../../../components/shared/Footer';

function setup() {
  const wrapper = shallow(<Footer />);

  return {
    wrapper
  };
}

describe('Footer', () => {
  it('should render with right amount of elements', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
