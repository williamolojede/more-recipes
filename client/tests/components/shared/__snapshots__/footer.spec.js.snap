// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render with right amount of elements 1`] = `
<footer
  className="page-footer"
>
  <div
    className="container"
  >
    <p>
      Crafted with 
      <i
        className="mdi mdi-heart"
      />
       by 
      <a
        href="http://william.ng/"
        rel="noopener noreferrer"
        target="_blank"
      >
        William I. Olojede
      </a>
    </p>
  </div>
</footer>
`;
